.github/workflows/prettier.yml

```diff
@@ -0,0 +1,40 @@
+name: Write prettier changes
+on:
+  push:
+    branches:
+      - master
+      - react
+
+jobs:
+  prettier:
+    strategy:
+      matrix:
+        node: [ 18 ]
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Setup Node
+        uses: actions/setup-node@v3
+        with:
+          node-version: ${{ matrix.node }}
+
+      - name: Cache dependencies
+        id: cache
+        uses: actions/cache@v3
+        with:
+          path: '**/node_modules'
+          key: ${{ matrix.node }}-${{ runner.os }}-modules-${{ hashFiles('**/yarn.lock') }}
+
+      - name: Install dependencies
+        run: yarn --ignore-engines
+
+      - name: Run prettier
+        run: yarn format
+
+      - name: Push to github
+        uses: mikeal/publish-to-github-action@master
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          BRANCH_NAME: react
```

Comment:
This will push to react branch when we commit to master. Please fix.

Reply:
good catch